README.md

```diff
@@ -47,6 +47,80 @@ This demo contains below applications.
 
 ### Automation of Applications in OpenShift.
 
+**Build and deploy with pipelines**
+
+***Install OpenShift Container Platform 3.5 in [CDK 3.0](https://developers.redhat.com/products/cdk/overview/)***
+
+Download the git repository by either forking, or simply cloning it. 
+
+```
+git clone https://github.com/myeung18/IntegrationApp-Automation.git
+```
+Start up your OpenShift environment by running
+
+```
+minishift start --username <USERNAME> --password <PASSWORD>
+oc login -u developer
+```
+
+Setup `rh-dev`, `rh-test` and `rh-prod` OpenShift projects as the CICD target environment (you may skip this step if you already have the environment ready, and this script will first delete the original projects in OpenShift and create new ones).
+    
+```
+./setup/setup.sh
+```
+
+Import the pipeline templates into your CICD project. For this case, it is `rh-dev`.
+
+```
+#switch to rh-dev project
+oc project rh-dev
+
+#import fisuser-service pipeline
+oc process -f pipelinetemplates/pipeline-appmodule-build.yml -p MODULE_NAME=fisuser-service -p IMAGEREGISTRY=172.30.1.1:5000 | oc create -f - 
+
+#import maingateway-service pipeline
+oc process -f pipelinetemplates/pipeline-appmodule-build.yml -p MODULE_NAME=maingateway-service -p IMAGEREGISTRY=172.30.1.1:5000 | oc create -f - 
+
+#import nodejsalert-ui pipeline
+oc process -f pipelinetemplates/pipeline-appmodule-build.yml -p MODULE_NAME=nodejsalert-ui -p IMAGEREGISTRY=172.30.1.1:5000 | oc create -f - 
+
+#import fisalert-service pipeline
+oc process -f pipelinetemplates/pipeline-appmodule-build.yml -p MODULE_NAME=fisalert-service -p IMAGEREGISTRY=172.30.1.1:5000 | oc create -f - 
+
+#import integration-master-pipeline
+oc process -f pipelinetemplates/pipeline-app-build.yml -p IMAGEREGISTRY=172.30.1.1:5000 | oc create -f -
+```
```

Comment:
I don't know if this is just me, cause there are a lot of people who use the same syntax as you've used here, but IMO, you should do oc new-app instead, unless you want to save the processed template for later or something. For example: oc new-app -f pipelinetemplates/pipeline-appmodule-build.yml -p MODULE_NAME=fisalert-service -p IMAGEREGISTRY=172.30.1.1:5000

No need to process template and pipe it into create.

Reply:
Good to know 'oc new-app' could do the same and is the better practice. I will give it a try and change accordingly.